pkg/command/root.go

```diff
@@ -42,3 +45,57 @@ func Execute() error {
 
 	return app.Run(os.Args)
 }
+
+// NewLogger initializes a service-specific logger instance.
+func NewLogger(cfg *config.Config) log.Logger {
+	return log.NewLogger(
+		log.Name("phoenix"),
+		log.Level("info"),
+		log.Pretty(true),
+		log.Color(true),
+	)
+}
+
+// ParseConfig loads proxy configuration from Viper known paths.
+func ParseConfig(c *cli.Context, cfg *config.Config) error {
+	// ParseConfig loads proxy configuration from Viper known paths.
+	logger := NewLogger(cfg)
+
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	viper.SetEnvPrefix("OCIS_ACCOUNTS")
```

Comment:
How is our naming scheme?
in proxy we have `PROXY_HTTP_ADDR`

Could we avoid naming collisions with other projects?

Reply:
I'd prefix it with `ACCOUNTS`

Reply:
by prefixing the extension name we avoid collisions